kubernetes/kubernetes.go

```diff
@@ -123,6 +123,11 @@ const rbacError string = `RBAC role in the Cloudability namespace may need to be
 // CollectKubeMetrics Collects metrics from Kubernetes on a predetermined interval
 // nolint: gocyclo
 func CollectKubeMetrics(config KubeAgentConfig) {
+	isInvalidClusterName, _ := regexp.MatchString(`[^a-zA-Z\d-]`, config.ClusterName)
```

Comment:
This check is too specific, currently the only case that can cause problems is a string containing only whitespace.

This would break uploads for many clusters that are otherwise functional.

Reply:
Yep, I mistakenly thought it was supposed to be following the k8s cluster conventions. Changed the logic just to trim whitespace and see if it's an empty string after